src/components/HappHosting/HappHosting.connector.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { MockedProvider } from 'react-apollo/test-utils'
+import wait from 'waait'
+import AllHappsQuery from 'graphql/AllHappsQuery.gql'
+import connector from './HappHosting.connector'
+
+const mocks = [
+  {
+    request: {
+      query: AllHappsQuery
+    },
+    result: {
+      data: {
+        allHapps: [
+          {
+            id: 1,
+            title: 'Holofuel',
+            thumbnailUrl: 'thumb.png',
+            homepageUrl: 'home.com',
+            hash: 'hash1'
+          },
+          {
+            id: 2,
+            title: 'Holo Community',
+            thumbnailUrl: 'thumb.png',
+            homepageUrl: 'home.com',
+            hash: 'hash2'
+          }
+        ]
+      }
+    }
+  }
+]
+
+describe('connector', () => {
+  it('runs the AllHapsQuery', async () => {
```

Comment:
Misleading test name - we're checking the results and not if the query actually executed :P

Reply:
hmm. debatable. Actually all we care about here is that the query ran. So what's misleading is all those `expects`, we probably just want one.

Reply:
Then I would *expect* :P `toHaveBeenCalled` in expect ;)